src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -13,6 +13,8 @@
      * <p>Это задание необходимо решать с использованием логических операторов.</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        boolean counter = (a ? 1 : 0) + (b ? 1 : 0) + (c ? 1 : 0) + (d ? 1 : 0) == 2;
```

Comment:
Inline this variable with return statement. There is "yellow" hint at Intellij Idea